community-stabilization-fund-api/pages/api/form-responses/reset-form-responses.ts

```diff
@@ -0,0 +1,16 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import { executeQuery, queries } from '../../../src/db';
+
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+const resetFormHandler = async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
We don't need a separate handler for formResponses. Put the method in the formResponsesHandler with the rest of the methods.